src/verify/VerificationStartScreen.tsx

```diff
@@ -32,9 +32,33 @@ import { waitUntilSagasFinishLoading } from 'src/redux/sagas'
 import colors from 'src/styles/colors'
 import fontStyles from 'src/styles/fonts'
 import { Spacing } from 'src/styles/styles'
-import { getPhoneNumberState } from 'src/verify/utils'
 import { walletAddressSelector } from 'src/web3/selectors'
 
+function getPhoneNumberState(
```

Comment:
this was moved (no changes were made) from the utils file - i removed the utils file since this is the only function left there